lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -54,3 +58,18 @@ def pre_save(self, obj):
         # There is nothing to pre-save. The default behavior changes the Course.id attribute from
         # a CourseKey to a string, which is not desired.
         pass
+
+
+class OrderView(APIView):
+    """ Retrieve order detailst. """
+
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):  # pylint:disable=unused-argument
```

Comment:
Why does this take `args` and `kwargs`, rather than just having an explicit argument for `number`?